bamt/builders/evo_builder.py

```diff
@@ -76,6 +77,10 @@ def __init__(
         self.default_max_arity = 100
         self.default_max_depth = 100
         self.default_timeout = 180
+        self.default_num_of_generations = 50
+        self.default_early_stopping_iterations = 50
+        self.verbose = (True,)
```

Comment:
What is tuple here for?

Reply:
That's what black did. It's going to make it tuple in any case